diff -urN v0.5.0/retroshare-nogui/src/retroshare-nogui.pro v0.5.0.new/retroshare-nogui/src/retroshare-nogui.pro
--- v0.5.0/retroshare-nogui/src/retroshare-nogui.pro	2010-06-30 01:47:04.740638978 +0200
+++ v0.5.0.new/retroshare-nogui/src/retroshare-nogui.pro	2010-06-30 04:06:40.151647563 +0200
@@ -1,4 +1,5 @@
 TEMPLATE = app
+CONFIG -= qt
 TARGET = retroshare-nogui
 
 ################################# Linux ##########################################
@@ -6,14 +7,21 @@
 	#CONFIG += version_detail_bash_script
 	QMAKE_CXXFLAGS *= -D_FILE_OFFSET_BITS=64
 
+	isEmpty(PREFIX) {
+		PREFIX = /usr
+	}
+
 	system(which gpgme-config >/dev/null 2>&1) {
 		INCLUDEPATH += $$system(gpgme-config --cflags | sed -e "s/-I//g")
 	} else {
 		message(Could not find gpgme-config on your system, assuming gpgme.h is in /usr/include)
 	}
 
-	LIBS += ../../libretroshare/src/lib/libretroshare.a
-	LIBS += -lssl -lgpgme -lupnp 
+	LIBS += $${PREFIX}/lib/libretroshare.a
+	LIBS += -lssl -lgpgme -lupnp
+
+	target.path = $${PREFIX}/bin
+	INSTALLS *= target
 }
 
 linux-g++ {
